### ISSUE TITLE
Align message bottom label to the right side of the bubble (incoming message)

### ISSUE THREAD

**Issue by maxxx777** [MessageKit/MessageKit#826]
Just an idea for future enhancements, to align message bottom label to the right side of the bubble (incoming messages).

![example_align_right](https://user-images.githubusercontent.com/2142832/44521838-cb5dcd80-a6d4-11e8-9ca2-9aa9ffeda02d.jpeg)

Of course, it is possible to use custom cell (put message text and time text together), but would be nice to align message bottom label (and message top label) accordingly to the width of the bubble.

**Comment by SD10** [comment]
This should already be possible by changing the label alignment properties on a cell's respective `MessageSizeCalculator` object

**Comment by maxxx777** [comment]
yes, it is possible for top label if we change `incomingCellTopLabelAlignment`, 
but there is no option for bottom label like `incomingCellBottomLabelAlignment`.
If we change `incomingMessageBottomLabelAlignment` to `.right`, it aligns the label to the right side of the collection view (not the right side of the bubble):

![example_align_right_2](https://user-images.githubusercontent.com/2142832/44649700-e1c19d00-a9e4-11e8-90d8-a13afafd159a.jpeg)

Should I create a new issue to add `incomingCellBottomLabelAlignment` and `outgoingCellBottomLabelAlignment` to `MessageSizeCalculator`?

**Comment by SD10** [comment]
@maxxx777 IIRC, the reason why this isn't possible is for performance reasons. The size of the cell labels surrounding the `MessageLabel` are static. When you change the `incomingCellBottomLabel` alignment what you are really doing is changing the `NSTextAlignment` for that label. You can achieve the effect you want by setting the insets for that label as well. 

However, I don't think this will have support in MessageKit beyond what is capable already

**Comment by maxxx777** [comment]
> You can achieve the effect you want by setting the insets for that label as well.

yes, but there should be different insets for each cell, because of different width of bubbles.

@SD10 thanks for your time. I guess I can close the issue.

**Comment by SD10** [comment]
@maxxx777 That's a fair point. I'll look into this a bit more 👍
